src/MspFcComms.cpp

```diff
@@ -60,6 +60,16 @@ namespace FcComms
     }
 
     // Send the rc commands to the FC using the member array of rc values.
+    FcCommsReturns MspFcComms::getRawRC()
+    {
+        MSP_RC msp_getrawRC;
+        msp_getrawRC.getRc(rc_values); // rc_values array initialized in corresponding .hpp file
+        #pragma GCC warning "Handle return"
+        sendMessage(msp_getrawRC);
+        ROS_INFO("rc_values are %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d, %d", rc_values[0],  rc_values[1],  rc_values[2],  rc_values[3],  rc_values[4],
+        rc_values[5],  rc_values[6],  rc_values[7],  rc_values[8],  rc_values[9], rc_values[10],  rc_values[11],  rc_values[12],  rc_values[13],  rc_values[14], rc_values[15],  rc_values[16]);
+        //Look at getstatus for a todo regarding ROS::debug
```

Comment:
None of the other getValue functions print out their results. Maybe it would be better to call this function from somewhere and print out the values there.